connect/transforms/src/main/java/org/apache/kafka/connect/transforms/util/Requirements.java

```diff
@@ -40,13 +40,27 @@ public static void requireSchema(Schema schema, String purpose) {
         return (Map<String, Object>) value;
     }
 
+    public static Map<String, Object> requireMapOrNull(Object value, String purpose) {
+        if (value == null) {
```

Comment:
I don't mind them but some people claim the ternary operator hurts readability and I don't care much one way or the other, so I tend to avoid it except in cases where not using it actually makes the code even less clear (e.g. the cases earlier in this patch where extra blocks of code in between two logically coherent lines of code would make it harder to understand).